Modules/RatMama.py

```diff
@@ -0,0 +1,206 @@
+async def handle_ratmama_announcement(ctx: Context) -> None:
+
+    """
+    Handles the Announcement made by RatMama.
+    Details are extracted, wrapped in a Rescue object and appended to the Rescue board.
+    An appropriate answer will be sent to IRC.
+
+    Args:
+        ctx: Context of the announcement
+
+    Returns: None
+
+    """
+
+    if ctx.user.nickname.casefold() not in (
+            k.casefold() for k in config.config["ratsignal_parser"]["announcer_nicks"]
+    ):
+        return
+
+    message: str = ctx.words_eol[0]
+    result = re.fullmatch(ratmama_regex, message)
+    client_name: str = result.group("cmdr")
+    system_name: str = result.group("system")
+    platform_name: str = result.group("platform")
+    o2_status: bool = result.group("o2") == "OK"  # false is CR
+    lang_code: str = result.group("language_code")
+    nickname: Optional[str] = result.group("nick")
+
+    exist_rescue: Rescue = ctx.bot.board.find_by_name(client_name)
+
+    if exist_rescue:
+        # we got a case already!
+        await ctx.reply(f"{client_name} has reconnected! Case #{exist_rescue.board_index}")
+        # now let's make it more visible if stuff changed
+        diff_response: str = ""
+        if system_name.casefold() != exist_rescue.system.casefold():
+            diff_response += f"System changed! "
+
+        if platform_name.casefold() != exist_rescue.platform.name.casefold():
+            diff_response += "Platform changed! "
+
+        if not o2_status != exist_rescue.code_red:
+            diff_response += "O2 Status changed!" if o2_status else\
+                "O2 Status changed, it is now CODE RED!"
+
+        if diff_response:
+            await ctx.reply(diff_response)
+
+    else:
+        platform = None
+
+        if platform_name.casefold() in ("pc", "ps", "xb"):
+            platform = Platforms[platform_name.upper()]
+        else:
+            LOG.warning(f"Got unknown platform from {ctx.user.nickname}: {platform_name}")
+
+        # no case for that name, we have to make our own
+        rescue: Rescue = Rescue(client=client_name, system=system_name, irc_nickname=nickname,
+                                code_red=not o2_status, lang_id=lang_code, platform=platform)
+
+        ctx.bot.board.append(rescue, overwrite=False)
+        index = ctx.bot.board.find_by_name(client=client_name).board_index
+        await ctx.reply(f"RATSIGNAL - CMDR {client_name} - "
+                        f"Reported System: {system_name} (distance to be implemented) - "
+                        f"Platform: {platform_name} - "
+                        f"O2: {'OK' if o2_status else 'NOT OK'} - "
+                        f"Language: {result.group('full_language')}"
+                        f" (Case #{index}) ({platform_name.upper()}_SIGNAL)"
+                        )
+
+
+@rule(r"\bratsignal\b", case_sensitive=False, full_message=True, pass_match=False, prefixless=True)
+async def handle_selfissued_ratsignal(ctx: Context) -> None:
```

Comment:
This name is a little confusing. when i see `selfissued` in the context of mecha, it makes it sound like this method only handles when mecha itself raises one. 

Thus, i suggest refactor/renaming this method to be more descriptive.
something to the tune of `handle_ratsignal` should do the trick.